internal/io/http/rest_sink.go

```diff
@@ -56,14 +56,33 @@ func (me MultiErrors) Error() string {
 func (ms *RestSink) Collect(ctx api.StreamContext, item interface{}) error {
 	logger := ctx.GetLogger()
 	logger.Debugf("rest sink receive %s", item)
-	resp, err := ms.Send(ctx, item, logger)
+	decodedData, _, err := ctx.TransformOutput(item)
```

Comment:
Move this out may have problem if "Send" is used elsewhere

Reply:
i searched the code and did not see other usage , can we make it a private function ?

Reply:
That's fine if no one use it